tools/gocti_type_generator/pyproject.toml

```diff
@@ -10,10 +10,10 @@ authors = [
 description = "Generator tool for GoCTI."
 readme = "README.md"
 license = {text  = "Apache 2.0"}
-version = "0.27.0"
+version = "0.28.0"
 dependencies = [
     "python-dotenv",
-    "pycti==6.6.8",
+    "pycti==6.6.10",
```

Comment:
Why are we jumping from `6.6.8` to `6.6.10` ?  I think that even if there was no graphql changes in `6.6.9` we should use the corresponding version of pycti

Reply:
The generation is only done with the last version of OpenCTI, if we want an update for `6.6.9` we need to do it manually. But there were no graphql changes and the goal is to only make a new release when we have graphql changes (or diff in their generated file `opencti-platform/opencti-graphql/src/generated/graphql.ts`)

Reply:
ok